### PR TITLE
make vault responsive

### PR DESCRIPTION
**Pull request by DanHillesheim**
## Type of change

- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Making the vault responsive and updating the ciphers view. Needs review and sign-off. 

## Code changes

This is mostly a change/edit/addition of bootstrap styles to the components to create a responsive vault experience.

## Testing requirements

Was building without warnings/errors until merging master into this branch. Need some help figuring out what's happening.

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Comment by DanHillesheim** [comment]
No errors on build, just warnings. Build seemingly completes ok, but localhost won't load. There are still items that need to be fixed in this commit, but without a working build I can't finish them.

**Comment by DanHillesheim** [comment]
Seems like it was an id10t error. It is running fine now. Thanks to @djsmith85 for helping me recognize that.

File: src/scss/variables.scss [hunk 1]
```diff
@@ -69,11 +69,11 @@ $btn-font-weight: 600;
...
-  xs: 0,
-  sm: 1px,
-  md: 2px,
-  lg: 3px,
-  xl: 4px,
+  xs: 300px,
+  sm: 640px,
+  md: 960px,
+  lg: 1200px,
+  xl: 1600px,
```
Comment (Hinton) [inline src/scss/variables.scss:76]: These seems kind of big. Have we considered using the bootstrap default values?

https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L195-L201

**Comment by Hinton** [comment]
@DanHillesheim it starts fine for me. Although it looks quite off in many places.

File: src/scss/variables.scss (hunk 1 above)
Comment (DanHillesheim) [reply src/scss/variables.scss:76]: We can look at other sizing options, but I felt it worked well in this case with the vault items.

**Comment by Hinton** [comment]
Closing due to move to mono repo

**Pull request closed without merging**